functests/dpdk/dpdk.go

```diff
@@ -575,7 +629,7 @@ func CreateSriovNetwork(sriovDevice *sriovv1.InterfaceExt, sriovNetworkName stri
 
 }
 
-func CreateDPDKWorkload() (*corev1.Pod, error) {
+func CreateDPDKWorkload(profile *perfv1alpha1.PerformanceProfile) (*corev1.Pod, error) {
```

Comment:
I would pass directly the node selector to the pod, instead of the perf profile.
This should also take into accont an eventual NODE_SELECTOR specified from outside.

Reply:
> I would pass directly the node selector to the pod, instead of the perf profile.
Done
> This should also take into accont an eventual NODE_SELECTOR specified from outside.
Not part of this PR. Please create a story, we should add this accross all our tests.